client/calypso-stubs/state/data-layer/wpcom-http.js

```diff
@@ -0,0 +1,7 @@
+/**
```

Comment:
This file should be `calypso-stubs/state/data-layer/wpcom-http/index.js`, same path as in Calypso. If works anyway, but it's confusing.